EMML/8001-9000/EMML8509.xml

```diff
@@ -0,0 +1,553 @@
+<note>Indication for reading: <foreign xml:lang="gez">ዘአብርሃም፡</foreign></note>
+<textLang mainLang="gez"></textLang>
+
+<colophon xml:id="coloph2">
+<locus target="155r"/>
+ስራዩ፡ ወአስተሰርዩ፡ በእንቲኣነ፡ ወተዘከሩ፡ በጸሎተክሙ፡ ወጸልዩ፡ ከመ፡ ይክፍለነ፡ መክፍልቶሙ፡ ለቅዱሳን፡ ሰማዕት፡ ዘገድፍኩ፡ ወዘወሰኩ፡ ሰረዩ፡ ሊተ፡ ወተዝክሩኒ፡ በጽሎተክሙ፡ ወባርኩኒ፡ ወበርከተክሙ፡
+ ይብጽሐኒ፡ ሰረዩ፡ ሊተ። አነ፡ ገብርከሙ፡ ሕኢቅየብስ።
+<note>The colophon is transcribed and translated in <bibl><ptr target="bm:Bausi2022Colophons"/><citedRange unit="page">136</citedRange></bibl>.</note>
+</colophon>
+
+</msItem>
+
+<msItem xml:id="ms_i50">
+<locus from="155r" to="162r"/>
+<title type="complete" xml:lang="en" ref="LIT6493Dersan">Homily on the Virgin Mary by Yoḥannǝs, bishop of Ethiopia</title>
+<note>Indication for reading: <foreign xml:lang="gez">በማሪያም፡</foreign></note>
+<textLang mainLang="gez"></textLang>
+
+<colophon xml:id="coloph3">
+<locus from="161v" to="162r"/>
+ወተፈጽመ፡ ድርሳን፡ ዘብፅዕት፡ ማሪያም፡ ቅድስት፡ ድንግል፡ እሙ፡ ለእግዚእነ፡ ዘደረሰ፡ አባ፡ ዮሐንስ፡ ጳጳስ፡ ዘአይቶጲያ፡ ዘአንበበ፡ ዘሰምዕ፡ ብሉ፡ ይምሕሮሙ፡ እግዚአ፡ ብሔር፡ ወይስርይ፡ ሎሙ፡ ኃጢአቶሙ፡ አሜን።
+</colophon>
+
+</msItem>
+
+<msItem xml:id="ms_i51">
+<locus from="162v" to="164r"/>
+<title type="complete" xml:lang="en" ref="LIT6909HomilyAstona">Homily by ʾAstona concerning Sunday</title>
+<note>Indication for reading: <foreign xml:lang="gez">በሰንበት፡ ክርስቲያን፡</foreign></note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i52">
+<locus from="164r" to="171v"/>
+<title type="complete" xml:lang="en" ref="LIT1269Dersan">Homily by the Orthodox on the Sabbaths</title>
+<note>Indication for reading: <foreign xml:lang="gez">በሰንበት፡ ክርስቲያን፡</foreign></note>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i53">
+<locus from="171v" to="172r"/>
+<title type="incomplete" xml:lang="en" ref="LIT1812GospelLuke">Beginning of the Gospel of Luke. The texts ends abruptly at 1:26.</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+
+</msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+  <supportDesc>
+    <support>
+      <material key="parchment"/>
+    </support>
+    <extent>
+      <measure unit="leaf">172</measure>
+      <dimensions type="outer" unit="cm">
+        <height>40.5</height>
+        <width>30.5</width>
+      </dimensions>
+    </extent>
+
+    <foliation>
+    <list>
+
+      <item xml:id="fol1">
+         European foliation in pencil at the bottom margin of the folio.
+      </item>
+
+    </list>
+    </foliation>
+
+
+    <condition key="good"></condition>
+  </supportDesc>
+  <layoutDesc>
+    <layout columns="2" writtenLines="33" cert="high"/>
+    <layout>
+<ab type="coronis">The beginnings of homilies are marked by a coronis.</ab>
+<ab type="cruces">A chi-rho sign on <locus target="102r">f. 102r</locus>.</ab>
+
+    </layout>
+  </layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+<handNote script="Ethiopic" xml:id="h1">
+<desc>The manuscript was not used by Uhlig in his "Äthiopische Paläographie". It shows many palaeographic peculiarities e.g.
```

Comment:
If I remember it right, all the mentioned features are not described at all in Uhligs palaeography. If this is the case, I would suggest to write rather:
`The manuscript shows many palaeographic peculiarities not mentioned in <bibl><ptr target="bm:Uhlig1988Palaographie"/></bibl>, e.g.`
However, I would not start with the information, that this manuscript was not used by Uhlig, because he took a small selection for his monograph and I would suggest to encode the work with a bibl.